apps/moodytunes/views.py

```diff
@@ -34,3 +43,30 @@ class EmotionPlaylistsView(FormView):
 
     def get_form_instance(self):
         return self.form_class(user=self.request.user)
+
+
+@method_decorator(login_required, name='dispatch')
+class SuggestSongView(FormView):
+    template_name = 'suggest.html'
+    form_class = SuggestSongForm
+
+    def post(self, request, *args, **kwargs):
```

Comment:
We should add some rate-limiting functionality to this view. We don't want users slamming our task worker at best, and also don't want one user to fill up our system with songs of their choice. Something like 3 POST requests per second sounds reasonable.

[django-ratelimit](https://django-ratelimit.readthedocs.io/en/latest/index.html) seems like a library we can use for this purpose